django-server/server/api/urls.py

```diff
@@ -26,6 +26,11 @@
 urlpatterns = [
     path('', include(router.urls)),
     path('', include(cafeterias_router.urls)),
+    path('cafeterias/<int:cafeteria_pk>/stats/hour', HourStatsView.as_view()),
+    path('cafeterias/<int:cafeteria_pk>/stats/day', DayStatsView.as_view()),
+    path('cafeterias/<int:cafeteria_pk>/stats/week', WeekStatsView.as_view()),
+    path('cafeterias/<int:cafeteria_pk>/stats/month', MonthStatsView.as_view()),
+    path('cafeterias/<int:cafeteria_pk>/stats/year', YearStatsView.as_view()),
```

Comment:
`cafeterias/<int:cafeteria_pk>/occupancy/` with grouping through filters. Once we implement other filters there will be 5 copies of everything.

Reply:
Written in more abstract way now, only the interface has to be implemented and neme specified in single place to get stats for all types (they don't have to specified explicitly for every stats type)